redbot/cogs/mod/mod.py

```diff
@@ -113,6 +111,16 @@ def cog_unload(self):
             ).format(prefix=prefix)
             self.bot.loop.create_task(self.bot.send_to_owners(msg))
             await self.settings.version.set(__version__)
+        if await self.settings.version() < "1.2.0":
+            prefixes = await self.bot.get_valid_prefixes()
+            prefix = re.sub(rf"<@!?{self.bot.user.id}>", f"@{self.bot.user.name}", prefixes[0])
+            msg = _(
+                "Delete delay settings have been moved. "
+                "Please use `{prefix}movedeletedelay` if "
+                "you were previously using these functions."
+            ).format(prefix=prefix)
+            self.bot.loop.create_task(self.bot.send_to_owners(msg))
+            await self.settings.version.set(__version__)
```

Comment:
This code should now use helper function `send_to_owners_with_prefix_replaced` as the part above this fragment (it doesn't show in diff, but if you look at code on `V3/develop`, it will be there).

I also noticed that both this message and the message for `[p]moveignoredchannels` gets sent on fresh Red install, it would be great to avoid it.

Reply:
Unfortunately the best solution for removing this message on a fresh install is setting a depracation time on the message say at version 3.x we just remove the message hoping people who want to migrate settings from before this verision they have already.

Reply:
We could loop through all guild settings and only send the message if `delete_delay` has been changed from the default in at least one of them.